Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM debian:stretch
-RUN apt-get update && apt-get install -y nodejs nodejs-legacy git wget build-essential python libcairo2-dev libgif-dev libpango1.0-dev libjpeg62-turbo-dev apt-transport-https redis-server libmapnik-dev && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y nodejs nodejs-legacy git wget python apt-transport-https redis-server libboost-filesystem-dev libboost-program-options-dev libboost-regex-dev libboost-system-dev libboost-thread-dev libxml2 libxml2-dev libfreetype6 libfreetype6-dev libjpeg-dev libpng-dev libtiff5-dev libcairo2 libcairo2-dev ttf-unifont ttf-dejavu ttf-dejavu-core ttf-dejavu-extra build-essential libgdal-dev libharfbuzz-bin libharfbuzz-dev libmapnik-dev && rm -rf /var/lib/apt/lists/*
```

Comment:
Same comment as on https://github.com/kartotherian/kartotherian/pull/73.  Many of these need not be explicitly installed.

Reply:
Done.